pkg/ingester/ingester_test.go

```diff
@@ -1409,8 +1409,8 @@ func BenchmarkIngesterPush(b *testing.B) {
 	require.NoError(b, err)
 
 	const (
-		series  = 10000
-		samples = 10
+		series  = 10
```

Comment:
Last time I checked, the average we see at Grafana Labs was around 300. I would increase it.

Reply:
Are you thinking of distributors?  I did 
```
sum (rate(cortex_ingester_ingested_samples_total[5m])) / 
sum (rate(cortex_request_duration_seconds_count{route="/cortex.Ingester/Push",status_code="success"}[5m]))
```

and got 8.4.